src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -36,6 +37,26 @@
  */
 final class ClassAttributesSeparationFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface, WhitespacesAwareFixerInterface
 {
+    /**
+     * @internal
+     */
+    const SPACING_NONE = 'none';
```

Comment:
More efficient to make this `false` and the other constant `true`. Strings are slower than booleans behind the scenes.

Reply:
Mh how do you see this for the user? Do you expect the config to change? If not, how do you see it being used internally? Thanks!

Reply:
Strings are slower than bools at runtime. This change will just make fixing faster, and won't affect usage.

Reply:
It will affect usage: those string constants are used to build the fixer configuration. Using booleans would require to translante string configuration options. Not worth it IMO, I'd keep the current implementation.

Reply:
@GrahamCampbell 
After discussing it with @ruudk I can make a config transformer that takes the user input (`one` or `none`) and converts it into booleans and use that internally? 

I'm willing to change the code, but first I need to know what exactly is asked. I don' think we should change the usage for the user as I think that `one` or `none` is much more clear than true/false.

Reply:
I think this micro optimization is not needed, I prefer the current setup, so please don't change it :)